app/src/main/java/com/infomaniak/drive/ui/menu/GalleryViewModel.kt

```diff
@@ -36,28 +36,46 @@ class GalleryViewModel : ViewModel() {
     private var currentCursor: String? = null
 
     val galleryApiResult = MutableLiveData<Pair<ArrayList<File>, IsComplete>?>()
+    private var lastGalleryFiles = ArrayList<File>()
+
+    val needToRestoreFiles get() = galleryApiResult.isInitialized
```

Comment:
One thing I like to do is to always group public variables with each other and so on.